### DISCUSSION TITLE
Automate application submission review

### DISCUSSION THREAD

**Discussion by MaximeAF** [codemagic-ci-cd/codemagic-docs#1882] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

Start a build on code magic
Publish the build on IOS and Android stores
Configure the new app release
Submit the release for review

### Expected results

Hello, I need to automate the submission process on Apple and Google stores. Currently, I manage several applications on the same codemagic account. Each application has its own  custom description. Currently, we have automated the publishing action from code magic to the stores, but then I need to submit the application review manually. During this step, I set the description to specify the new min app version. So my question is, is it possible to automate the submit review process on code magic with a user interface allowing me to specify an app version number, a description and a release note customized for each app ?  For information, I currently launch builds for each application one by one. For example, if I have app1 and app2, I run 4 builds. 2 runs for app1 on IOS and Android, then 2 runs for app2 on IOS and Android.

### Actual results

I submit my applications for review mannualy.

**Comment by himesh-cm** [comment]
Hi @MaximeAF, There is no user interface that allows you to follow the steps you mentioned via UI.
You can set up scripts and use codemagic API to trigger builds.
We have documentation [here](https://docs.codemagic.io/yaml-quick-start/white-label-getting-started/) on how you can achieve the desired result.

I recommend you to [connect](https://codemagic.io/contact/) with our solutions engineering team and discuss best possible approach for your use case.

**Reply by MaximeAF** [reply]
Thank you for this answer @himesh-cm

**Reply by himesh-cm** [reply]
Happy to help.
While I’m closing our discussion for now, if you have any other questions or suggestions, feel free to reply back or start a new discussion anytime. Have a nice day!
